util/reggen/uvm_reg_base.sv.tpl

```diff
@@ -56,13 +86,24 @@ ${make_ral_pkg_window_class(dv_base_prefix, esc_if_name, window)}
     // registers
   % for r in rb.all_regs:
 <%
-      r0 = r.reg if isinstance(r, MultiRegister) else r
+      regs = []
+      if isinstance(r, MultiRegister):
+        if r.dv_compact:
+          regs.append(r.reg)
+        else:
+          regs.extend(r.regs)
+      else:
+        regs.append(r)
```

Comment:
As above for the definition of `regs` here.

Reply:
Done